pkg/experimental/experimental.go

```diff
@@ -0,0 +1,26 @@
+package experimental
+
+const (
+	// BuildDrivers experimental flag
+	BuildDrivers = "build-drivers"
+)
+
+// FeatureFlags is an enum of possible feature flags
+type FeatureFlags int
+
+const (
+	// FlagBuildDrivers indicates if configurable build drivers are enabled.
+	FlagBuildDrivers FeatureFlags = iota + 1
+)
+
+// ParseFlags converts a list of feature flag names into a bit map for faster lookups.
+func ParseFlags(flags []string) FeatureFlags {
```

Comment:
Neat, I'll have to keep this technique in mind.

Reply:
I could have converted to a map and then used key lookups, but what is nice about this is that it's faster than a map lookup and uses less space too.